sample/kotlin/src/main/kotlin/com/datadog/android/sample/sessionreplay/ImageComponentsFragment.kt

```diff
@@ -61,42 +67,55 @@ internal class ImageComponentsFragment : Fragment() {
         loadImageButtonBackground()
     }
 
+    @OptIn(ExperimentalRumApi::class)
     private fun loadImageView() {
         viewModel.fetchRemoteImage(
             LARGE_IMAGE_URL,
             imageViewRemote,
             object : ImageLoadedCallback {
                 override fun onImageLoaded(resource: Drawable) {
                     imageViewRemote.setImageDrawable(resource)
+                    if (imageLoadedCounter.decrementAndGet() == 0) {
```

Comment:
should we extract this into a common private method? same snippet is used 4 times.